docs/en/enterprise-edition/rule-reference/iac-security/storage/appsec-azure-2.adoc

```diff
@@ -9,7 +9,7 @@
 |Prisma Cloud Policy ID 
 | 564c3bcd-7b29-4e6a-9da9-e929876a9f1f
 
-|Checkov ID 
```

Comment:
I'm aware of this issue. I've noted it, and will fix it in the review cycle.

Root cause: the metadata table wasn't rewritten/updated because the script couldn't find the "Policy Details" section. It exists, not as a heading, but as bolded para text. This is the only instance of a non-existent "Policy Details" section in the doc set. I'll update the mainline source, and that will fix this issue.